conf/messages

```diff
@@ -250,7 +250,7 @@ tariff.declaration.item.commodityDetails.common.linkText.0 = Data Element 6/14 -
 tariff.declaration.item.commodityDetails.common.linkText.1 = Data Element 6/8 - Description of Goods (opens in new tab)
 tariff.declaration.item.unDangerousGoodsCode.common.linkText.0 = Data Element 6/12 - UN Dangerous Goods Code (opens in new tab)
 tariff.declaration.item.cusCode.common.linkText.0 = Data Element 6/13 - CUS Code for Chemicals (opens in new tab)
-tariff.declaration.item.additionalTaricCode.common.linkText.0 = Data Element 6/16 - Commodity Code: TARIC Additional Code (opens in new tab)
+tariff.declaration.item.nationalAdditionalCode.common.text = This entry will complete<br>{0}.
```

Comment:
If you wouldn't mind removing this line? Post CEDS-5317, the default tariff expander text is "For details, go to {0}", which is now being automatically added if the '_.common.text' is not there to override.